### PR TITLE
Changed clear command

### PR DESCRIPTION
**Pull request by jonushzw**
Now clear command is clear address book to separate address and wedding book

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ClearWeddingBookCommand.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import seedu.address.model.Model;
+import seedu.address.model.WeddingBook;
+
+/**
+ * Clears the address book.
+ */
+public class ClearWeddingBookCommand extends Command {
+
+    public static final String COMMAND_WORD = "clear-weddingbook";
```
Comment (zzibo) [inline src/main/java/seedu/address/logic/commands/ClearWeddingBookCommand.java:13]: can consider abstracting the clear command into an interface/ abstract class like how we plan to implement add and tag

**Review by zzibo — COMMENTED** [review]
LGTM! will merge after we decide on the oop implementation

File: src/test/java/seedu/address/logic/commands/ClearWeddingBookCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import static seedu.address.testutil.TypicalWeddings.getTypicalWeddingBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.WeddingBook;
+
+public class ClearWeddingBookCommandTest {
```
Comment (ywllowsensor) [inline src/test/java/seedu/address/logic/commands/ClearWeddingBookCommandTest.java:14]: Good use of test cases!

**Review by ywllowsensor — COMMENTED** [review]
Overall, LGTM. This function will be useful for the target user!

**Pull request merged**